### PR TITLE
Added Bugsnex.Plug to the Readme

### PR DESCRIPTION
**Pull request by bitboxer**

### CODE REVIEW

**Comment by PragTob** [comment]
Hi there :wave: 

Thanks for the contribution, would you be willing to change it up a bit? The plug is optional and fits better into the `Usage` section imo after the more general `Sending bug reports` section :)

Cheers + thanks,
Tobi

**Comment by bitboxer** [comment]
Are you sure? I think most people will not read that far and will wonder why it is not working. Maybe I can change the wording a bit to say that this is optional?

**Comment by PragTob** [comment]
I'm pretty sure. If the section says "Usage" in bold big letters and people can't be bothered to read it I really can't help - that's the most important one should read imo, after configuration (because installation should be clear). Especially if things are not working :)

We could maybe clean up the README a bit to make it a bit more structured, but I like that all use cases are in the Usage section. Not everyone needs the plug as well (our first application didn't use it in a long time)

If you feel strongly about it I'm ok with adding something like "Check out how to send notifications to bugsnag via `Bugsnag.notify` or the plug in the "Usage" section" although it being the very next session starting after the sentence might be a bit of an overkill.

**Comment by bitboxer** [comment]
Not feeling super strong about it. Will change  as you suggested with a sentence as 3rd point that hints on further configuration that is needed

**Comment by bitboxer** [comment]
Updated the pr

File: README.md [hunk 1]
```diff
@@ -8,16 +8,22 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
 
   1. Add bugsnex to your list of dependencies in `mix.exs`:
 
-        def deps do
-          [{:bugsnex, "~> 0.0.1"}]
-        end
+     ```elixir
+     def deps do
+       [{:bugsnex, "~> 0.2.1"}]
```
Comment (PragTob) [inline README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -32,22 +38,34 @@ config :bugsnex, :use_logger, true
...
 You can also manually wrap code in a `Bugsnex.handle_error` block. Errors in this block will then be sent to Bugsnag and reraised. Example:
 
-
 ```elixir
 Bugsnex.handle_errors %{some: "metadata"} do
   somthing_that_could_raise_and_error()
 end
 ```
 
+Bugsnex also provides a Plug called `Bugsnex.Plug` that you could add to
```
Comment (PragTob) [inline README.md]: I think the plug is important enough to warrant its own `### Automatic reports via Bugsnex.Plug` or something more beautiful than that heading. It's a minor thing, I can also add it manually :)

**Pull request merged**